maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -616,8 +608,6 @@ public void clean( ReleaseCleanRequest cleanRequest ) throws ReleaseFailureExcep
                 ( (ResourceGenerator) phase ).clean( cleanRequest.getReactorProjects() );
             }
         }
-
-        updateListener( cleanRequest.getReleaseManagerListener(), "cleanup", PHASE_END );
```

Comment:
Why is this information not necessary anymore?

Reply:
same

Reply:
Agreed.